.github/deploy.md

```
@@ -0,0 +1,28 @@
+**AWS_REGION**  
+The AWS region where your resources (ECR repository) reside (for example, `us-east-1`).
+
+---
+
+**ECR_REGISTRY**  
+The full URL to your Amazon ECR registry (for example, `123456789012.dkr.ecr.us-east-1.amazonaws.com`).
+
+---
+
+**ECR_REPOSITORY**  
+The specific name of the repository in ECR where your Docker images should be pushed (for example, `my-php-app`).
```

Markdownlint (0.37.0)

Files should end with a single newline character

(MD047, single-trailing-newline)